README.md

```diff
@@ -50,8 +51,9 @@ Then follow the C++ instructions to [add a module from the Viam Registry](https:
 
 These instructions assume a target system similar to Debian Bullseye.
 
-Before you can build your module's executable, you must [create your module locally](https://docs.viam.com/registry/create/#prepare-the-module-for-execution), following the C++ instructions. 
-Once your module is created, follow the steps to [prepare the module for execution](https://docs.viam.com/registry/create/#prepare-the-module-for-execution).
+Before you can build this module's executable, you must follow the steps to [prepare a module for execution](https://docs.viam.com/registry/create/#prepare-the-module-for-execution).
+
+You can also view the [complex C++ module examples](https://github.com/viamrobotics/viam-cpp-sdk/tree/main/src/viam/examples/modules/complex) to see how a simple module like this one can be modified. 
```

Comment:
(minor) I would update this language slightly. I think "to see how a simple module like this one can be modified" implies that the complex module is structured as a "let's start with something simple and then iterate on it step by step" which isn't quite right. It is true that the complex module does show ways to modify/add to a module! Perhaps you could modify to say, "to see an example of how one could structure and create APIs for a more extensible module."? Or something to that effect.

Reply:
Great point. I might suggest removing this sentence entirely, actually: this readme is wholly this-module-specific, the broader conceptual intro to modules and their levels of complexity is more the responsibility of the main docs pages under `/registry/`.

Or, if desired, perhaps mention could live at the very end, maybe phrased similarly to:

> For a more involved C++ module example, which defines both a component and service API, see the [C++ complex module example](https://github.com/viamrobotics/viam-cpp-sdk/tree/main/src/viam/examples/modules/complex).

Reply:
Agreed, thank you both for your feedback!